SONIC-DEV/sonic-buildimage
# [platform/cel] Update Fishbone32 and Phalanx i2c driver to fix write 0xFF

**- What I did**
Update the Celestica platform submodule pointer for fixing 0xFF written to optical module EEPROM.

**- How I did it**
Applied the following changes to Phalanx and Fishbone32:
1. Apply clock stretching workaround.
2. Set interrupt acknowledge bit every time sends a command to open-core CMD register.
3. Add delay 8 clocks before start polling TIP bit.

**- How to verify it**
Run the stress test script to check if there are two or more consecutive 0xFF returned from the optical module EEPROM dump result.
